pack/promo/tftbw.json

```diff
@@ -16,6 +16,7 @@
                 "text": [
                     "[Hh]eals? (that much )?(((\\d+|all|(X total)) )?damage (from that asset )?(and|or) )?((\\d+|all|(X total)) )?horror"
```

Comment:
will we delete the 'text' line once the tag is up and running?

Reply:
Eventually yeah, I wanted to leave it in place for people who were expecting the API.

The website will now ignores the 'text' field going forward though, so its value is only there for legacy api users.